gradle/libs.versions.toml

```diff
@@ -86,6 +86,9 @@ mockito = "4.11.0"
 # https://github.com/mockk/mockk/releases
 mockk = "1.13.7"
 
+# https://github.com/takahirom/roborazzi/releases
+roborazzi = "1.9.0-alpha-2"
```

Comment:
Where is the dependency from Roborazzi -> Robolectric? Ideally Roborazzi would be using public Android APIs to take screenshots and perform screendiffing.

Reply:
Great insights! Robolectric's integration tests, which include Espresso tests, have notable similarities to those in Roborazzi. Incorporating these tests could be beneficial.

## Whether Roborazzi has a dependency on Robolectric
Roborazzi -> Robolectric

I've configured Roborazzi to function independently from Robolectric from the start. Although Roborazzi utilizes the Espresso API, it scarcely uses the Robolectric API. I've verified that Roborazzi can compile without the compileOnly dependency, except for roborazzi-compose, which registers an Activity for user convenience. This is not a test target in this PR.
https://github.com/takahirom/roborazzi/blob/main/roborazzi-compose/src/main/java/com/github/takahirom/roborazzi/RoborazziCompose.kt#L62

Dependencies

Robolectric -> public Android APIs
:roborazzi -> Espresso -> public Android APIs
:roborazzi -> public Android APIs
:roborazzi-compose -> Robolectric


## Whether adding integration tests with Roborazzi into Robolectric implies a dependency of Robolectric on Roborazzi
Robolectric -> Roborazzi

This is merely an integration test, so it relies on Robolectric developers rather than end-users. I acknowledge this might be burdensome, which is why I stated, “Should this integration prove more burdensome than beneficial, we are open to reevaluating and possibly removing it to maintain the overall health and efficiency of the test suite."

Reply:
> I recommend to test code for latest version, instead of an old Roborazzi version.

Do you mean using '+' for dependencies? While this approach is effective, it could introduce a risk of Roborazzi breaking Robolectric's build at any time. Robolectric employs Dependabot for updating dependencies, which I believe is sufficient for maintaining updates.